lib/src/components/NxCheckbox/NxCheckbox.scss

```diff
@@ -4,13 +4,18 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
+@import '../../scss-shared/_nx-focus-border-helpers';
+
+$nx-checkbox__box-height: 16px;
+$nx-checkbox__box-width: 16px;
```

Comment:
Using BEM style for variable names isn't something we've done so far.  @glenhunter do you think that's something we want to introduce or should these be changed?

Reply:
> Using BEM style for variable names isn't something we've done so far. @glenhunter do you think that's something we want to introduce or should these be changed?

IIRC we decided a long time ago that variables would not use BEM. If we were going to change that it should have happened in the shift to CSS variables. So I say that they should be changed

Reply:
> > Using BEM style for variable names isn't something we've done so far. @glenhunter do you think that's something we want to introduce or should these be changed?
> 
> IIRC we decided a long time ago that variables would not use BEM. If we were going to change that it should have happened in the shift to CSS variables. So I say that they should be changed

I understand now that for something as trivial as this, we do not require BEM. 
5b2f6050d4210671e331277e019a6cc41f19475f